worldedit-core/src/main/java/com/fastasyncworldedit/core/util/TextureUtil.java

```diff
@@ -367,14 +362,14 @@ public TextureUtil(File folder) throws FileNotFoundException {
                     LOGGER.info("Asset jar down has been downloaded successfully.");
                 } catch (IOException e) {
                     LOGGER.error(
-                        "Could not download version jar. Please do so manually by creating a `FastAsyncWorldEdit/textures` folder with `.minecraft/versions` jar in it.");
+                            "Could not download version jar. Please do so manually by creating a `FastAsyncWorldEdit/textures` folder with `.minecraft/versions` jar in it.");
                     LOGGER.error("If the file exists, please make sure the server has read access to the directory.");
                 }
             } catch (AccessControlException e) {
                 LOGGER.error(
-                    "Could not download asset jar. It's likely your file permission are setup improperly and do not allow fetching data from the Mojang servers.");
+                        "Could not download asset jar. It's likely your file permission are setup improperly and do not allow fetching data from the Mojang servers.");
                 LOGGER.error(
-                    "Please create the following folder manually: `FastAsyncWorldEdit/textures` with `.minecraft/versions` jar in it.");
+                        "Please create the following folder manually: `FastAsyncWorldEdit/textures` with `.minecraft/versions` jar in it.");
```

Comment:
Update instructions with new path. Also asset jar > version jar. We should stick to one term.